Sources/Overlays/_Testing_CoreGraphics/Attachments/Attachment+AttachableAsCGImage.swift

```diff
@@ -23,46 +25,9 @@ extension Attachment {
   ///     to a test report or to disk. If `nil`, the testing library attempts
   ///     to derive a reasonable filename for the attached value.
   ///   - contentType: The image format with which to encode `attachableValue`.
```

Comment:
Consider documenting the default `nil` value, and saying "if any" etc.

Reply:
`nil` is described in the prose below. "if any" doesn't make sense here since there will be an image format one way or another.